script/script.js

```diff
@@ -0,0 +1,24 @@
+// Declare input and output
+const ioChatbot = {
+   input : "hello",
+   output : "hi"
+};
+console.log(ioChatbot);
+
+// Reply function for getting input element
+function reply(){
+   const question = document.querySelector("#input");
+   if(question.value === "hello"){
```

Comment:
Instead of comparing `question.value` with the string `"hello"`, you should compare `question.value` with the `input` of your object, `ioChatbot`.